internal/pkg/template/templates/workloads/partials/cf/taskrole.yml

```diff
@@ -102,5 +102,23 @@ TaskRole:
                 - !Ref {{logicalIDSafe $topic.Name}}SNSTopic
               {{- end}}
       {{- end}}{{- end}}
-
-
+      {{- if eq .Observability.Tracing "AWSXRAY"}}
+      - PolicyName: 'AWSDistroOpenTelemetryPolicy' 
+        PolicyDocument:
+          Version: '2012-10-17'
+          Statement:
+            - Effect: 'Allow'
+              Action:
+                - 'logs:PutLogEvents'
+                - 'logs:CreateLogGroup'
+                - 'logs:CreateLogStream'
+                - 'logs:DescribeLogStreams'
+                - 'logs:DescribeLogGroups'
+                - 'xray:PutTraceSegments'
+                - 'xray:PutTelemetryRecords'
+                - 'xray:GetSamplingRules'
+                - 'xray:GetSamplingTargets'
+                - 'xray:GetSamplingStatisticSummaries'
+                - 'ssm:GetParameters'
+              Resource: "*"
```

Comment:
Out of curiosity, how did you know which actions to permit here?
I understand that for broader otel we might need additional permissions:
https://aws-otel.github.io/docs/setup/ecs/create-iam-policy but I was curious if we needed `ssm:GetParameters` if all this integration can be used for is "xray"?

Reply:
I got the policy from [here](https://aws-otel.github.io/docs/setup/permissions).

As far as I can tell, the only use for `ssm:GetParameters` is to get a custom configuration file, so you're right - we don't need it for this PR, where all we support is the default config. If we want to support custom config in the future, we'll have to add it then.

I think if we added support for custom configuration, and people wanted to start using that feature, they'd have to redeploy their service _anyways_, so changing the policy in the future should be ok - given that, do you think it makes sense to remove the `ssm:GetParameters` action for now?

Reply:
Talked offline. We're removing it. 😊